features/custom_stack_trace.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Custom stack trace
+
+  Scenario: Error.prepareStackTrace override
+    Given a file named "features/a.feature" with:
+      """
+      Feature: Some feature
+        Scenario: Some scenario
+          Given I override Error.prepareStackTrace
+      """
+    Given a file named "features/step_definitions/cucumber_steps.js" with:
+      """
+      var cucumberSteps = function() {
+        this.When(/^I override Error.prepareStackTrace$/, function() {
+          Error.prepareStackTrace = function() {
+            return 'Custom message';
+          }
```

Comment:
@charlierudolph my bad, it failed for me but because of the missing `✓ ` which I didn't see initially. I've just pushed an update for the test which this time I made sure were giving the error on master:
```
(...)/node_modules/cucumber/lib/cucumber/support_code/library.js:53
        var line = stackframes[1].getLineNumber();
```